examples/AI-surrogate-mode-hard-magnets.ipynb

```diff
@@ -0,0 +1,384 @@
+    "\n",
+    "Details about the models and their training data can be found in the [mammos-ai documentation](https://mammos-project.github.io/mammos/api/mammos_ai.html).\n",
+    "\n",
+    "This approach allows for rapid predictions of hysteresis properties based on the intrinsic micromagnetic parameters computed earlier.\n",
+    "\n",
+    "The model has been trained on a 50 x 50 x 50 nm cubic grain with uniaxial anisotropy and applied field applied along the cube's edge."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "id": "4e143880",
+   "metadata": {
+    "editable": true,
+    "slideshow": {
+     "slide_type": ""
+    },
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "T = np.linspace(0, 1.1 * results_kuzmin.Tc.q, 20)\n",
+    "results_ml = mammos_ai.Hc_Mr_BHmax_from_Ms_A_K(\n",
+    "    Ms=results_kuzmin.Ms(T),\n",
+    "    A=results_kuzmin.A(T),\n",
+    "    Ku=results_kuzmin.K1(T),\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "id": "343e020a",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "ExtrinsicProperties(Hc=Entity(ontology_label='CoercivityHcExternal', value=array([2942256.2  , 2857558.5  , 2740996.8  , 2543269.5  , 2428668.5  ,\n",
+       "       2208944.   , 2115859.   , 1819307.6  , 1808106.2  , 1698485.6  ,\n",
+       "       1829058.5  , 1429938.1  , 1091347.4  ,  903446.3  , 1116855.1  ,\n",
+       "       1057900.2  ,  966576.75 ,   17457.65 ,   17204.068,   17204.068],\n",
+       "      dtype=float32), unit='A / m'), Mr=Entity(ontology_label='Remanence', value=array([1142329.1  , 1137940.8  , 1132234.9  , 1115691.5  , 1108781.   ,\n",
+       "       1087777.1  , 1069930.6  , 1039830.7  ,  997795.75 ,  947130.56 ,\n",
+       "        841426.94 ,  782085.25 ,  755020.2  ,  736731.56 ,  618404.6  ,\n",
+       "        582873.56 ,  390627.44 ,   61490.266,   57923.465,   57923.465],\n",
+       "      dtype=float32), unit='A / m'), BHmax=Entity(ontology_label='MaximumEnergyProduct', value=array([409878.44, 406728.28, 402647.75, 390956.53, 386122.5 , 371605.25,\n",
+       "       359498.5 , 339515.28, 312623.38, 281647.78, 222328.2 , 192068.8 ,\n",
+       "       178941.03, 170193.77, 119876.34, 106301.82,  47868.95, 349298.25,\n",
+       "       350828.28, 350828.28], dtype=float32), unit='J / m3'))"
+      ]
+     },
+     "execution_count": 7,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "results_ml"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "0ccb6bcc",
+   "metadata": {},
+   "source": [
+    "This has returned an `ExtrinsicProperties` object containing the predicted hysteresis properties. This is the same object type as returned by running micromagnetic simulations and them processing them with `mammos_analysis` in the hard magnet tutorial.\n",
```

Comment:
I would swap the first paragraph and cell 7 that shows the repr of `results_ml`.